test/cri-containerd/policy_test.go

```diff
@@ -16,6 +16,7 @@ import (
 	"time"
 
 	"github.com/Microsoft/hcsshim/internal/shimdiag"
+	securityPolicyTest "github.com/Microsoft/hcsshim/test/pkg/securitypolicy"
```

Comment:
does vscode not complain about this? i think the go preference is to have all lowercase in package names?

Reply:
I use goland and it doesn't lol. I can change.

Reply:
if the linter passes, its file lol